SQL Exception does not cause abnormal exit

I got an Athena error with this plugin, but it didn't result in an abnormal exit.
Below are some of the logs from that time.
```
2020-09-04 09:05:58.886 +0000 [INFO] (0015:task-0000): Loading 30,690 rows
2020-09-04 09:06:03.408 +0000 [INFO] (0015:task-0000): > 4.52 seconds (loaded 215,165 rows in total)
java.sql.SQLException: Error fetching results
 at com.amazonaws.athena.jdbc.AthenaResultSet.next(AthenaResultSet.java:184)
 at org.embulk.input.athena.AthenaInputPlugin.run(AthenaInputPlugin.java:133)
 at org.embulk.spi.util.Executors.process(Executors.java:62)
 at org.embulk.spi.util.Executors.process(Executors.java:38)
 at org.embulk.exec.LocalExecutorPlugin$DirectExecutor$1.call(LocalExecutorPlugin.java:170)
 at org.embulk.exec.LocalExecutorPlugin$DirectExecutor$1.call(LocalExecutorPlugin.java:167)
 at java.util.concurrent.FutureTask.run(FutureTask.java:266)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
 at java.lang.Thread.run(Thread.java:748)
Caused by: com.amazonaws.athena.jdbc.shaded.com.amazonaws.services.athena.model.AmazonAthenaException: Rate exceeded (Service: AmazonAthena; Status Code: 400; Error Code: ThrottlingException; Request ID: 09759443-002f-49a4-b411-971607667f03)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1588)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1258)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1030)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:742)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:716)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:699)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:667)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:649)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:513)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.services.athena.AmazonAthenaClient.doInvoke(AmazonAthenaClient.java:1549)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.services.athena.AmazonAthenaClient.invoke(AmazonAthenaClient.java:1525)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.services.athena.AmazonAthenaClient.executeGetQueryResults(AmazonAthenaClient.java:1064)
 at com.amazonaws.athena.jdbc.shaded.com.amazonaws.services.athena.AmazonAthenaClient.getQueryResults(AmazonAthenaClient.java:1040)
 at com.amazonaws.athena.jdbc.AthenaServiceClient.fetchQueryResult(AthenaServiceClient.java:205)
 at com.amazonaws.athena.jdbc.AthenaStatementClient.paginateQueryResult(AthenaStatementClient.java:264)
 at com.amazonaws.athena.jdbc.AthenaStatementClient.nextQueryResult(AthenaStatementClient.java:292)
 at com.amazonaws.athena.jdbc.AthenaResultSet.fetchRows(AthenaResultSet.java:133)
 at com.amazonaws.athena.jdbc.AthenaResultSet.next(AthenaResultSet.java:164)
 ... 9 more
2020-09-04 09:06:20.364 +0000 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
```
I'd like to create a workflow that triggers an abnormal exit of embulk and reruns it.
Is there any reason not to have an abnormal exit?

--- a/src/main/java/org/embulk/input/athena/AthenaInputPlugin.java
+++ b/src/main/java/org/embulk/input/athena/AthenaInputPlugin.java
@@ -142,6 +142,7 @@ public class AthenaInputPlugin implements InputPlugin
                         }
                         catch (SQLException e) {
                             e.printStackTrace();
+                            throw new RuntimeException(e);
                         }
                     }
 
@@ -153,6 +154,7 @@ public class AthenaInputPlugin implements InputPlugin
                         }
                         catch (SQLException e) {
                             e.printStackTrace();
+                            throw new RuntimeException(e);
                         }
                     }
 
@@ -170,6 +172,7 @@ public class AthenaInputPlugin implements InputPlugin
                         }
                         catch (SQLException e) {
                             e.printStackTrace();
+                            throw new RuntimeException(e);
                         }
                     }
 
@@ -187,6 +190,7 @@ public class AthenaInputPlugin implements InputPlugin
                         }
                         catch (SQLException e) {
                             e.printStackTrace();
+                            throw new RuntimeException(e);
                         }
                     }
 
@@ -204,6 +208,7 @@ public class AthenaInputPlugin implements InputPlugin
                         }
                         catch (SQLException e) {
                             e.printStackTrace();
+                            throw new RuntimeException(e);
                         }
                     }
 
@@ -224,6 +229,7 @@ public class AthenaInputPlugin implements InputPlugin
         }
         catch (Exception e) {
             e.printStackTrace();
+            throw new RuntimeException(e);
         }
         finally {
             try {

